src/svd2ir.rs

```diff
@@ -264,6 +264,11 @@ pub fn convert_svd(svd: &svd::Device) -> anyhow::Result<IR> {
     let mut ir = IR::new();
 
     let mut device = Device {
+        nvic_priority_bits: if let Some(cpu) = &svd.cpu {
```

Comment:
I think it'd make sense to make `nvic_priority_bits` an `Option<u8>`, so the amount of prio bits isn't known then the code is not emitted, instead of defaulting to 0. Also because the value doesn't apply to other archs like riscv.